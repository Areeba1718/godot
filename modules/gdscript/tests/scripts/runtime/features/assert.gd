func get_condition(x: bool) -> bool:
	print("get_condition(%s)" % x)
	return x

func get_message(x: bool) -> String:
	print("get_message(%s)" % x)
	return "message"

func test():
	assert(get_condition(false), get_message(false))
