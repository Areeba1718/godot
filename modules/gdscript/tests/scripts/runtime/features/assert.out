GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR
>> on function: test()
>> runtime/features/assert.gd
>> 10
>> Assertion failed: message
